mapper.go

```diff
@@ -655,6 +656,22 @@ func existingDirMapper(r *Registry) MapperFunc {
 	}
 }
 
+func ptrMapper(r *Registry) MapperFunc {
+	return func(ctx *DecodeContext, target reflect.Value) error {
+		elem := reflect.New(target.Type().Elem()).Elem()
+		nestedMapper := r.ForValue(elem)
+		if nestedMapper == nil {
+			return errors.New("cannot find member for pointer element type")
```

Comment:
Please include information about the nested type in the message, unless it's wrapped. Also are there tests for this case?

Reply:
Done.